codelabs/codelab-03/README.md

```diff
@@ -2,7 +2,7 @@
 
 ### Due Date
 
-This codelab comes in two parts, both of which are due on *Thursday, February 15th at 11:59:59PM*. This is codelab-03 (part B) which expands upon what was learned in [codelab-02](https://umd-cs-stics.gitbooks.io/cmsc389l-practical-cloud-computing-spring-2018/content/codelabs/codelab-02/) (part A) and introduces CloudFront.
+This is codelab 3, which expands upon ![codelab-02](../../../media/codelabs/codelab-02/README.md) by introducing CloudFront. Both are due on *Thursday, February 15th at 11:59:59PM*.
```

Comment:
Is this supposed to be an image link?

Did you mean: [codelab 2] (../codelab-02/README.md)?